### DISCUSSION TITLE
It would be nice if the collecting record listing showed the name of a single file

### DISCUSSION THREAD

**Discussion by dschissler** [pocketbase/pocketbase#1414] [Q&A]
It seems that a file field is represented as a string.  It would be nice if that could be displayed in the collection listing when only a single file is allowed in that field.

**Comment by ganigeorgiev** [comment]
You can see the name when you click on the file or open the edit panel.
I think it will clutter the UI if we print the name in the table (the name also has random part so I'm not sure how useful that would be).

**Reply by dschissler** [reply]
Right now there is a big empty space.  If I were to put a lot of static assets in this collection then it would be hell to browse it.  Also, couldn't you just do some magic to cut off the random file characters and then add the extension back?

![Screenshot from 2022-12-28 20-34-25](https://user-images.githubusercontent.com/3968727/209857422-f9ad0bb1-802c-4774-9a00-adf747b81b91.png)

**Reply by dschissler** [reply]
Maybe you could do some CSS magic that would de-prioritize this field if space is needed?

**Reply by dschissler** [reply]
I also noticed that you're not displaying the thumbnail at it's full 100x100.  Maybe it could have an over cursor effect to grow it to the full thumbnail size.

**Reply by ganigeorgiev** [reply]
Hm, I've added it in my todo to consider it eventually when applying the other planned Admin UI improvements in v0.12/v0.13.

**Reply by dschissler** [reply]
Another idea could be a special file list mode for a collection.
